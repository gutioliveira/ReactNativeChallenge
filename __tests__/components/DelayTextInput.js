import 'react-native';
import React from 'react';
import DelayTextInput from '../../app/components/delayTextInput';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {

    const tree = renderer.create(
        <DelayTextInput />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});


