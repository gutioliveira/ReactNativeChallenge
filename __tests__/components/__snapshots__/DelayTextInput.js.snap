// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  onChangeText={[Function]}
  pauseDelay={500}
/>
`;
